layouts/black-lives-matter.hbs

```diff
@@ -265,19 +265,18 @@
       community. It falls on the rest of us.
     </p>
     <p>
-      To Black folks looking for access or collaboration: We're here for you.
```

Comment:
(Didn't remove _collaboration_. Instead, relocated it below to replace _guidance and any sort of support_.)